esmvaltool/diag_scripts/ensclus/ens_anom.py

```diff
@@ -0,0 +1,165 @@
+"""Computation of ensemble anomalies based on a desired value."""
+
+# Standard packages
+import sys
+import os
+import numpy as np
+from scipy import stats
+
+
+def ens_anom(filenames, dir_output, name_outputs, varname, numens, season,
+             area, extreme):
+    """Ensemble anomalies.
+
+    Computation of the ensemble anomalies based on the desired value
+    from the input variable (it can be the percentile, mean, maximum, standard
+    deviation or trend)
+    OUTPUT: NetCDF files of ensemble mean of climatology, selected value and
+    anomaly maps.
+    """
+    # User-defined packages
+    from read_netcdf import read3Dncfield, save_N_2Dfields
+    from sel_season_area import sel_season, sel_area
+
+    print('**************************OUTPUT*******************************')
+    print('The name of the output files will be <variable>_{0}.txt'
+          .format(name_outputs))
+    print('Number of ensemble members: {0}'.format(numens))
+
+    # Reading the netCDF file of 3Dfield, for all the ensemble members
+    var_ens = []
+    for ens in range(numens):
+        ifile = filenames[ens]
+        # print('ENSEMBLE MEMBER %s' %ens)
+        var, varunits, lat, lon, dates, time_units = read3Dncfield(ifile)
+
+        # Convertion from kg m-2 s-1 to mm/day
+        if varunits == 'kg m-2 s-1':
+            var = var * 86400  # there are 86400 seconds in a day
+            varunitsnew = 'mm/day'
+        else:
+            varunitsnew = varunits
+
+        # Selecting a season (DJF,DJFM,NDJFM,JJA)
+        var_season, dates_season = sel_season(var, dates, season)
+
+        # Selecting only [latS-latN, lonW-lonE] box region
+        var_area, lat_area, lon_area = sel_area(lat, lon, var_season, area)
+
+        var_ens.append(var_area)
+
+    if varunitsnew == 'mm/day':
+        print('\nPrecipitation rate units are converted from kg m-2 s-1 \
+                to mm/day')
+
```

Comment:
Replace the two lines above by
```python
        print('\nPrecipitation rate units are converted from kg m-2 s-1'
              'to mm/day')
```
because the way it is done now with the backslashes inserts all the spaces from the next line into the text, which looks quite strange, and anyway using backslashes is not good practice. This also goes for all other strings on multiple lines.